RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -4671,6 +4671,18 @@ boolean have_workshed() {
 	return true;
 }
 
+int baseNCForcesToday()
```

Comment:
this is only used as `X > 0`. Given the "not important to calculate this properly", might it work better intent-wise as a "boolean anyNCForcesAvailableToday"?

Reply:
No, I think it'd be more useful in future if left with more functionality. And it should hardly be a trial to maintain - one line to add right next to another, each time another NC force is added (so once every couple of years).